pkg/document/values.go

```diff
@@ -180,6 +182,10 @@ func createValueRowsFromList(
 	// documented without descriptions
 	if hasDescription {
 		jsonableObject := convertHelmValuesToJsonable(values)
+		if blankContainerDefaults {
+			// Don't document complex objects
+			jsonableObject = ""
```

Comment:
Maybe make the default something more descriptive like `body omitted`

Reply:
I realize this needs to be language-friendly, so my proposal is to change to a string parameter containing the replacement string ("" to keep original behaviour)